sdk/modelsrepository/azure-iot-modelsrepository/src/samples/java/com/azure/iot/core/Main.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.iot.core;
+
+/**
+ * Entry point for running samples.
+ */
+public class Main {
+    static void main(String[] args) throws InterruptedException {
+
+        // DtmiConventions samples
+        DtmiConventionsSamples.IsValidDtmi();
```

Comment:
Why capital I in Is, where other methods are camelCased?

Reply:
The casing is wrong, fixing